README.md

```diff
@@ -27,65 +27,55 @@ interfaces in pure Python.
     src="https://mybinder.org/badge_logo.svg"/>
 </a>
 
-# Install Django IDOM
+# Quick Example
 
-```bash
-pip install django-idom
-```
+## `example_app/components.py`
 
-# Django Integration
+This is where you'll define your [IDOM](https://github.com/idom-team/idom) components. Ultimately though, you should
+feel free to organize your component modules you wish. Any components created will ultimately be referenced
+by Python dotted path in `your-template.html`.
 
-To integrate IDOM into your application you'll need to modify or add the following files to `your_project`:
+```python
+from idom import component, html
+from django_idom import IdomWebsocket
 
-```
-your_project/
-├── __init__.py
-├── asgi.py
-├── settings.py
-├── urls.py
-└── example_app/
-    ├── __init__.py
-    ├── components.py
-    ├── templates/
-    │   └── your-template.html
-    └── urls.py
-```
```

Comment:
I think we should retain something like this - it's nice to have a high level picture of what all the files are. Perhaps you could move this under the "Quick Example" section heading?